.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
configure your editor to add an empty line at the end of the file
please will attach rubocop.yml in this pr